### ISSUE TITLE
SameDiff: TF import burndown list

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#6231]
A burndown list of ops to implement and test:

- [ ] get_seed, set_seed
- [x] Eye
- [x] SequenceMask
- [x] reduce_logsumexp
- [x] SplitV
- [ ] Imag
- [ ] LinSpace
- [x] LogMatrixDeterminant
- [ ] MatrixBandPart
- [ ] Mod
- [ ] NthElement
- [ ] NonMaxSuppression
- [ ] random_shuffle
- [x] ScatterNd ops
- [x] Size
- [ ] TruncateMod
- [ ] truncated_normal
- [ ] UnsortedSegmentMax/Min/Mean/Sum/Prod/SqrtN
- [ ] boolean_mask
- [x] broadcast_dynamic_shape
- [x] broadcast_to
- [ ] count_up_to
- [x] meshgrid
- [x] reduce_any, reduce_all
- [x] tensordot
- [ ] timestamp
- [x] where
- [ ] more strided slice test cases (inc. edge cases)

NN module:
- [x] l2_normalize
- [x] AvgPool3D, MaxPool3D (failing tests - missing import mapping)
- [x] LRN
- [x] Conv1D
- [x] Conv2d, max/avg pool 2d
- [x] Conv2DTranspose
- [x] Conv3d
- [x] FusedBatchNorm
- [x] LeakyRelu
- [x] alpha_dropout (technically in contrib)
- [x] embedding_lookup
- [x] softmax_cross_entropy_with_logits/v2

Layer module:
- [x] AveragePooling1D, MaxPooling1D
- [x] Conv1D
- [x] Dense
- [x] Flatten
- [x] SeparableConv1D

Loss functions:
- [x] absolute_difference
- [x] cosine_distance
- [x] hinge_loss
- [x] huber_loss
- [x] log_loss
- [x] mean_squared_error
- [x] sigmoid_cross_entropy
- [x] softmax_cross_entropy
- [x] l2_loss
- [x] sparse_softmax_cross_entropy

Also need more tests for where/while/if etc.
Also need a pass on RNG distributions.

Also need: a way to specify a validation function for ops/values for TF import
i.e., can't exactly compare random values to TF values - but we *can* check distribution properties (mean/min/max etc)

**Comment by AlexDBlack** [comment]
Closing this: remaining ops are either tested or have open issues for them.
